content/en/docs/tasks/configure-pod-container/quality-service-pod.md

```diff
@@ -109,6 +120,11 @@ A Pod is given a QoS class of Burstable if:
 * The Pod does not meet the criteria for QoS class Guaranteed.
 * At least one Container in the Pod has a memory or CPU request or limit.
 
+Burstable pods have some resource guarantees based on the request, but do not require a specific limit. If a limit is
+not specified they default to a limit equivalent to the capacity of the node, which allows them to flexibly increase
+their resources if available. These pods are killed only after all Best-Effort pods are evicted, or when exceeding
```

Comment:
Please always spell "BestEffort" in the same way to avoid confusion.

```suggestion
their resources if available. These pods are killed only after all BestEffort pods are evicted, or when exceeding
```